modules/docs/mdx/10.0-UPGRADE-GUIDE.mdx

```diff
@@ -128,23 +117,24 @@ these sizes should change along with it. Since `px` are a fixed unit and do not
 
 ## Component Updates
 
-### Buttons
-
-We re-factored how we styled Buttons to use our `createStyles` utility function. We don't anticipate
-this as a breaking change but, there may be slight changes to visual test.
+### Button
 
 **PR:** [#2285](https://github.com/Workday/canvas-kit/pull/2285)
 
+We've refactored how we style buttons to use our `createStyles` utility function. We don't
+anticipate this being a breaking change in most cases, but there may be slight changes to visual
+tests.
+
 #### Button Icon Fill
 
-Icons will no longer be "filled" on toggle. This decision was made to not have the exisitng icon
-look so different in the toggled state.
+In an effort to soften the visual effect when a button is toggled, icons will no longer be "filled"
+on toggle.
```

Comment:
I may have misunderstood the intended change here. When I first read this, I thought it was an aesthetic choice to remove the fill on toggle; however, I wonder if this is actually a bug fix. If so, we'll need to revise this language.